src/TwigExtension.php

```diff
@@ -110,17 +118,24 @@ public function renderUri(
      * Usage: {{ url('article_show', {'id': '3'}, {'foo': 'bar'}, 'fragment') }}
      * Generates: http://example.com/article/3?foo=bar#fragment
      *
-     * @param array $options Can have the following keys:
-     *     - reuse_result_params (bool): indicates if the current
-     *       RouteResult parameters will be used, defaults to true
+     * @param null|string $route
+     * @param array       $routeParams
+     * @param array       $queryParams
+     * @param null|string $fragmentIdentifier
+     * @param array       $options Can have the following keys:
+     *                             - reuse_result_params (bool): indicates if the current
+     *                             RouteResult parameters will be used, defaults to true
+     *
+     * @return string
```

Comment:
Please revert the changes to this docblock.

We only document parameters and return values that:

- do not have typehints, and/or
- have additional narrative descriptions.

Only `$options` falls under that category here.

Reply:
resolved.